x-pack/plugins/security_solution/server/endpoint/lib/artifacts/lists.test.ts

```diff
@@ -376,29 +381,29 @@ describe('artifacts lists', () => {
         .mockReturnValueOnce(first)
         .mockReturnValueOnce(second);
 
-      const resp = await getFilteredEndpointExceptionList({
+      const resp = await getFilteredEndpointExceptionListRaw({
         elClient: mockExceptionClient,
-        schemaVersion: 'v1',
         filter: TEST_FILTER,
         listId: ENDPOINT_LIST_ID,
       });
+      const translated = convertExceptionsToEndpointFormat(resp, 'v1');
 
-      // Expect 2 exceptions, the first two calls returned the same exception list items
-      expect(resp.entries.length).toEqual(2);
+      // Expect 1 exceptions, the first two calls returned the same exception list items
+      expect(translated.entries.length).toEqual(1);
```

Comment:
This change is due to a bug fix (I hope) from the old version.
Before changes, it was doing deduplication on each exceptions request call, now it is done after all exceptions have been requested, so the deduplication is over all the exceptions, not on each pagination. Let me know if this was done on this way before for any specific reason I missed.